namespaces.go

```diff
@@ -28,9 +32,13 @@ type NamespacesService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/namespaces.html
 type Namespace struct {
-	ID   int    `json:"id"`
-	Path string `json:"path"`
-	Kind string `json:"kind"`
+	ID                          int    `json:"id"`
+	Name                        string `json:"name"`
+	Path                        string `json:"path"`
+	Kind                        string `json:"kind"`
+	FullPath                    string `json:"full_path"`
+	ParentID                    int    `json:"parent_id"`
```

Comment:
It looks like `ParentID` should be a string:

![image](https://user-images.githubusercontent.com/4171547/34248135-2c88dab6-e635-11e7-903a-9ca0500f7c81.png)